### ISSUE TITLE
List of publications

### ISSUE THREAD

**Issue by t-neumann** [posquit0/Awesome-CV#38]
Hi,

great CV template, I like it a lot. Very well structured, too.

The only thing missing (or I couldn't find it) is a publications section where one could e.g. import bibTeX files. Is there any workaround for that?

Would be much appreciated,

Cheers

**Comment by johannesbottcher** [comment]
Use BibTeX as you would do in *any* other document. Nothing special here.

**Comment by t-neumann** [comment]
I tried that with

\bibliography{MyBibliography}
\nocite{*}

and of course that works, but the formatting does not match the CV template at all. Do I need to put it in any of the defined cv tags to apply the Awesome-CV style?

**Comment by johannesbottcher** [comment]
So the question is not how to get a bibliography into your cv, but how you can have a bibliography taat matches the rest of the document? 
Well, that is something @posquit0 deals with.

**Comment by t-neumann** [comment]
Yes sorry for being unclear - I edited the issue title.

**Comment by posquit0** [comment]
Yup, I have a plan to include bibTeX as a default. I'll update it ASAP.

These days, I'm spending so busy time... So, please wait more time or try it yourself now.

Good luck!

**Comment by t-neumann** [comment]
Great, this will be a very helpful feature for everyone. I'm still a LaTeX-Beginner, so I'll wait then.

**Comment by aminmkhan** [comment]
@t-neumann, you can also check posquit0/Awesome-CV#23 for a hack to include publications, in the meantime. It still doesn't matches Awesome-CV style, but the output is better formatted.

**Comment by t-neumann** [comment]
Hi @aminmkhan I pulled your changes but cannot get the example to run. It keeps saying "empty bibliography". I'm using xelatex - any ideas?

**Comment by xmlu** [comment]
Have you run `biblatex`?

**Comment by t-neumann** [comment]
@xmlu Switching to `biber` solved it - thanks!

**Comment by t-neumann** [comment]
Thanks to @aminmkhan for the hack - it's very elegantly integrated into the template code. The only thing left for me to really switch to this template is to also get the formatting right "Awesome-CV"-style. Now it just doesn't fit into the rest of the CV style. Is there any progress?

**Comment by aminmkhan** [comment]
@t-neumann You could try [this update](https://github.com/aminmkhan/Awesome-CV/pull/4).

**Comment by smarru** [comment]
@aminmkhan I am wondering if you have plans to merge your publications update with the main repo?

**Comment by aminmkhan** [comment]
I don't have admin access to merge to `master`, and like many other pull requests #23 is also left open.

For now, you can pull from my fork at:
https://github.com/aminmkhan/Awesome-CV/tree/feature/publications-biblatex
And also report issues and pull requests there related to publications section.
